engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/SpelValues.scala

```diff
@@ -2,38 +2,39 @@ package pl.touk.nussknacker.engine.flink.table
 
 object SpelValues {
 
-  val spelBoolean        = "true"
-  val spelStr            = "'str'"
-  val spelByte           = "123.byteValue"
-  val spelShort          = "123.shortValue"
-  val spelInt            = "123"
-  val spelFloat          = "123.12.floatValue"
-  val spelDouble         = "123.12.doubleValue"
-  val spelBigint         = "123.longValue"
-  val spelDecimal        = "T(java.math.BigDecimal).ONE"
+  val spelBoolean = "true"
+  val spelStr     = "'str'"
+  val spelChar    = "'1'.charAt(0)"
+
+  val spelByte    = "123.byteValue"
+  val spelShort   = "123.shortValue"
+  val spelInt     = "123"
+  val spelFloat   = "123.12.floatValue"
+  val spelDouble  = "123.12.doubleValue"
+  val spelBigInt  = "T(java.math.BigInteger).ONE"
```

Comment:
imo despite the misleading name `long` java type was the valid type here for DataTypes.BIGINT column.

Reply:
Yeah, you are right. I've been cheated by variable names. Now the name is as the java type, not a table API type.